src/lib/firebase.lib.ts

```diff
@@ -90,3 +95,10 @@ export function getSharedCampaignTracksCollection(campaignId: string) {
     constructSharedCampaignTracksDocPath(campaignId)
   ) as DocumentReference<TracksDocument>;
 }
+
+export function getUsersDoc(userId: string) {
```

Comment:
Nicely done here! I like that you've matched the existing structure for how I've been handling this perfectly.